mix.exs

```diff
@@ -1,12 +1,13 @@
 defmodule Snap.MixProject do
   use Mix.Project
 
-  @github_url "https://github.com/breakroom/snap"
-  @version "0.10.0"
+  @original_github_url "https://github.com/breakroom/snap"
+  @github_url "https://github.com/surgeventures/snap"
+  @version "0.10.1"
 
   def project do
     [
-      app: :snap,
+      app: :snap_fresha,
```

Comment:
I don't think you planned to commit this file :)

Reply:
Yeah, I needed it to publish a forked version of the package :) I will fix this and open a new PR later.